content/800-data-platform/100-accelerate/100-what-is-accelerate.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: 'What is Accelerate'
+metaTitle: 'What is Accelerate'
+metaDescription: 'Learn about Accelerate, a global cache and serverless connection pool that makes your database queries faster.'
+tocDepth: 3
+toc: true
+---
+
+<TopBlock>
+
+[Accelerate](https://www.prisma.io/data-platform/accelerate) is a global database cache available in 300 locations that you can use to achieve up to 1000x faster database queries.
+
+It provides easy access to a globally distributed cache and scalable connection pool for serverless applications via a powerful API. The goal of Accelerate is to improve response times and reduce database load. It works by caching data at the edge using established caching patterns that you control.
```

Comment:
@nikolasburk as Accelerate allows Prisma to be used on the edge right now, isn't that a feature aswell?

Reply:
good point, I adjusted my suggestion 👍  (it's not really a feature because it "falls out" of the connection pool, but I still think it's worth highlighting this explicitly)